domino_generator/example/ex1/ex1.g.dart

```diff
@@ -75,8 +90,38 @@ void renderEx2(_i0.DomContext $d) {
   $d.open('div');
   {
     String text_X_4b68ab38() =>
-        _i3.Intl.message('X', name: 'text_X_4b68ab38', args: [], desc: '');
+        ($strings[r'text_X_4b68ab38'].containsKey($d.globals['locale'])
```

Comment:
Simplify to:
`$strings[r'text_X_4b68ab38'][$d.locale]` (or `$d.globals.locale`). The context or the globals object could default it to empty string.